services/scabbard/libscabbard/src/migrations/diesel/sqlite/migrations/2022-06-16-131600-consensus-add-decision-ack/up.sql

```diff
@@ -0,0 +1,20 @@
+-- Copyright 2018-2022 Cargill Incorporated
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+-- -----------------------------------------------------------------------------
+
+ALTER TABLE consensus_2pc_context_participant
+  ADD COLUMN decision_ack Text NOT NULL DEFAULT 'FALSE' CHECK ( decision_ack IN ('TRUE', 'FALSE') );
```

Comment:
In sqlite, the correct storage type for boolean is Integer. Keywords TRUE and FALSE in sqlite are aliases for 1 and 0. See https://www.sqlite.org/datatype3.html section 2.1.

Reply:
The correct type may be NUMERIC, as there is an alias BOOLEAN->NUMERIC, as seen on that page.

Reply:
Fixed